internal/driver/cli.go

```diff
@@ -95,7 +96,7 @@ func parseFlags(o *plugin.Options) (*source, []string, error) {
 	// Recognize first argument as an executable or buildid override.
 	if len(args) > 1 {
 		arg0 := args[0]
-		if file, err := o.Obj.Open(arg0, 0, ^uint64(0), 0); err == nil {
+		if file, err := o.Obj.Open(arg0, 0, ^uint64(0), 0); err == nil || err == elf.ErrNoSymbols {
```

Comment:
Maybe handle this error at https://github.com/google/pprof/blob/b51a6544410de613fff320b4c2fd7411db01d474/internal/binutils/binutils.go#L324 where it actually happens? Letting the elf package error out seems like leaking the implementation details.